src/lib/y2network/widgets/wireless_eap_mode.rb

```diff
@@ -0,0 +1,48 @@
+require "cwm/common_widgets"
+
+module Y2Network
+  module Widgets
+    # Widget to select EAP mode.
+    class WirelessEapMode < CWM::ComboBox
+      # @param settings [Y2network::InterfaceConfigBuilder]
+      def initialize(settings)
+        textdomain "network"
+        @settings = settings
+      end
+
+      def init
+        self.value = @settings.eap_mode
+      end
+
+      def store
+        @settings.eap_mode = value
+      end
+
+      def label
+        _("EAP &Mode")
+      end
+
+      # generate event when changed so higher level widget can change content
+      # @see Y2Network::Widgets::WirelessEap
+      def opt
+        [:notify]
+      end
+
+      def items
+        [
```

Comment:
Constant?

Reply:
well, issue with constant is that I need to define it with `N_(` and then each value modify with `_()` method, which looks too much additional work for me.